src/components/Patient/PatientIdentifierFilter.tsx

```
@@ -200,127 +217,167 @@ export default function PatientIdentifierFilter({
+
+    return null;
+  };
+
+  const selectorContent = (
+    <Command shouldFilter={false} className="border-none">
+      <div className="flex flex-col">
+        {allIdentifierConfigs.length > 2 ? (
+          <div className="p-2">
+            <label className="text-xs text-gray-600 mb-1.5 ml-1 block">
+              {t("search_by")}
+            </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)